.github/workflows/release.yml

```diff
@@ -179,3 +179,11 @@ jobs:
         if: github.event.inputs.dry_run == 'false'
         run: |
           npx -p @semantic-release/changelog -p @semantic-release/git -p @semantic-release/exec -p semantic-release semantic-release
+
+      - name: Create or update stable branch after release.
+        if: github.event.inputs.dry_run == 'false'
+        run: |
+          git fetch origin
+          git checkout -B stable
```

Comment:
why `-B` here ?

Reply:
we have already created the `stable` branch, but it's working no issue we can keep it.

Reply:
Got it will checkout locally